doc/man3/EVP_EncryptInit.pod

```diff
@@ -197,70 +197,132 @@ symmetric ciphers.
 
 The B<EVP_CIPHER> type is a structure for cipher method implementation.
 
-EVP_CIPHER_fetch() fetches the cipher implementation for the given
-B<algorithm> from any provider offering it, within the criteria given
-by the B<properties>.
+=over 4
+
+=item EVP_CIPHER_fetch()
+
+Fetches the cipher implementation for the given B<algorithm> from any provider
+offering it, within the criteria given by the B<properties>.
```

Comment:
Could you please replace the occurrences of `B<argument>` with `I<argument>`? It seems the file is wildly inconsistent in regards to this.

Reply:
Done - that is what happens when you block copy the contents of one file to another :)